node_modules/oae-core/filepreview/css/filepreview.css

```diff
@@ -58,39 +58,27 @@
     }
 }
 
-/* MediaCore preview */
-
-#filepreview-container #filepreview-mediacore,
+#filepreview-container #filepreview-video-container,
 #filepreview-container #filepreview-markdown-container {
     padding: 20px 0;
 }
 
-#filepreview-container #filepreview-mediacore iframe {
-    width: 100%;
-}
-
-.mobile #filepreview-container #filepreview-mediacore iframe {
-    width: 340px; /* iOS will need an explicit width for the iframe */
-}
-
-#filepreview-container #filepreview-mediacore.filepreview-mediacore-audio iframe {
-    height: 24px;
-}
+/* Video preview */
 
 @media (min-width: 780px) {
```

Comment:
In the screenshot above, you can see that the video controls exceed the edges of the comments widget on the left and the right. The left side of the video player should be aligned with the left edge of the user's profile picture and the right side of the video player should be aligned with the right edge of the comment box.